Mathlib/Analysis/InnerProductSpace/Dual.lean

```diff
@@ -69,6 +70,61 @@ variable {E}
 theorem toDualMap_apply {x y : E} : toDualMap 𝕜 E x y = ⟪x, y⟫ :=
   rfl
 
+section NullSubmodule
+
+open SeparationQuotient LinearMap
+
+variable (E)
+
```

Comment:
You don't want this to be explicit everywhere. Similarly, `𝕜` should only be explicit when the hypotheses don't mention `𝕜`, but the conclusion does (e.g., `inner_eq_zero_of_left`.
```suggestion
variable (E) in
```